pages/index.vue

```diff
@@ -520,6 +522,9 @@
       }
     },
     computed: {
+      requestName() {
+        return this.label.match(/^([^?]*)\??/)[1]
+      },
```

Comment:
Label will be a string which user decides, we don't want to enforce any pattern for it. Isn't it?